### PR TITLE
[IMP] account: usability improvements

### PR DESCRIPTION
**Pull request by oco-odoo**
Task 2346547

### CODE REVIEW

File: addons/account/views/account_move_views.xml [hunk 1]
```diff
@@ -1342,7 +1358,7 @@
             <field name="name">Bills</field>
             <field name="res_model">account.move</field>
             <field name="view_mode">tree,kanban,form</field>
-            <field name="view_id" ref="view_invoice_tree"/>
+            <field name="view_id" ref="view_in_invoice_tree"/>
```
Comment (smetl) [inline addons/account/views/account_move_views.xml:1361]: same in `action_move_in_refund_type` & `action_move_in_receipt_type`

**Review by smetl — CHANGES_REQUESTED** [review]

File: addons/account/views/account_move_views.xml (hunk 1 above)
Comment (oco-odoo) [reply addons/account/views/account_move_views.xml:1361]: @smetl That wasn't done for out invoices; you sure about that?

**Comment by smetl** [comment]
@robodoo delegate+

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**